index.d.ts

```diff
@@ -1,5 +1,8 @@
-import mcData from 'minecraft-data'
+import {IndexedData} from 'minecraft-data'
 
-declare module "prismarine-registry" {
-  export default function(mcVersion: string): ReturnType<typeof mcData>
-}
\ No newline at end of file
+declare function loader(mcVersion: string): IndexedData
+declare namespace loader {
+  export type Registry = IndexedData
```

Comment:
What's this?

Reply:
This type is the type that can be returned by the loader. It is exported to make using it easier